app/ts/McdaApp/Workspace/CurrentSubproblemContext/CurrentSubproblemContext.tsx

```diff
@@ -47,14 +45,12 @@ export function CurrentSubproblemContextProviderComponent({
   const [configuredRanges, setConfiguredRanges] = useState<
     Record<string, [number, number]>
   >({});
+  const [stepSizeByCriterion, setStepSizeByCriterion] = useState<
```

Comment:
i think we mostly use convention multiple name for collection so in this case `stepSizesByCriterion`